.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,24 @@
+name: spellcheck
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: check out code
+      uses: actions/checkout@v2
+
+    - name: install misspell
+      run: |
+        curl -L -o ./install-misspell.sh https://git.io/misspell
```

Comment:
Is there a less risky way to install this than `curl http://redirection-service | bash` ? :)

e.g. For autoformatting we "go get" a thing.

Reply:
There is option using

$ go get -u github.com/client9/misspell/cmd/misspell

but  don't think golang is part of default `ubuntu-latest` installation.